src/main/webapp/js/stats-loader.js

```diff
@@ -0,0 +1,23 @@
+// Fetch stats and display them in the page.
+function fetchStats(){
+    const url = '/stats';
+    fetch(url).then((response) => {
+      return response.json();
+    }).then((stats) => {
+      const statsContainer = document.getElementById('stats-container');
+      statsContainer.innerHTML = '';
+      const messageCountElement = buildStatElement('Message count: ' + stats.messageCount);
+      statsContainer.appendChild(messageCountElement);
+    });
+  }
```

Comment:
The indentation that you have here is a little confusing; it makes it look like the code below is still within 'fethStats'.